src/App.css

```diff
@@ -1,7 +1,242 @@
 body {
```

Comment:
Your style looks almost identical to the one shown in the Zeplin design.

However, one of the project requirements was Your final design should match Zeplin's design.
Therefore, we must stick to the requirement. Currently, your Bookstore app is missing pale grey background around your main app.

- [ ] Please add it.

Here is how your app looks like
<img width="1440" alt="Screen Shot 2021-11-06 at 02 17 45" src="https://user-images.githubusercontent.com/60043698/140579839-775dc474-80ab-47e7-aaab-dabe334a68ea.png">

Zeplin's design:
<img width="1440" alt="Screen Shot 2021-11-06 at 02 18 26" src="https://user-images.githubusercontent.com/60043698/140579929-bc8df930-3957-4135-adfd-fd09a7b78881.png">